web-config.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 web:
+  domain: null
+  basePath: null
```

Comment:
Per other comments, I've removed `domain` but left `basePath`, as some frameworks don't have a canonical way to declare a "context path" (Express is one).